### PR TITLE
L98: Requiring C++14 in gRPC Core/C++ Library

### PR DESCRIPTION
**Pull request by veblush**

### CODE REVIEW

**Comment by veblush** [comment]
@markdroth @ctiller PTAL. And would the proper approvers be you guys instead of a11r?

File: L98-requiring-cpp14.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Requiring C++14 in gRPC Core/C++ Library
+----
+* Author(s): veblush
+* Approver: a11r
```
Comment (markdroth) [inline L98-requiring-cpp14.md:4]: You can make me the approver.

File: L98-requiring-cpp14.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Requiring C++14 in gRPC Core/C++ Library
```
Comment (markdroth) [inline L98-requiring-cpp14.md:1]: Please add "L98: " to the start of this line.

**Review by markdroth — APPROVED** [review]

File: L98-requiring-cpp14.md (hunk 1 above)
Comment (veblush) [reply L98-requiring-cpp14.md]: Done.

File: L98-requiring-cpp14.md (hunk 2 above)
Comment (veblush) [reply L98-requiring-cpp14.md]: Done.

**Pull request merged**